test/integration/ssh/run_command_test.rb

```diff
@@ -17,23 +17,34 @@
   end
   let(:hab_conn) { Train.create(:habitat, transport_opts).connection }
 
-  describe 'when using defaults' do
-    it 'should be able to get the hab version' do
-      result = hab_conn.run_hab_cli('--version')
-      result.exit_status.must_equal 0
-      # "hab 0.77.0/20190301212334\n"
-      result.stdout.must_match %r{^hab\s+\d+\.\d+\.\d+/\d+\n}
+  describe 'run_hab_cli' do
+    describe 'when using defaults' do
+      it 'should be able to get the hab version' do
+        result = hab_conn.run_hab_cli('--version')
+        result.exit_status.must_equal 0
+        # "hab 0.77.0/20190301212334\n"
+        result.stdout.must_match %r{^hab\s+\d+\.\d+\.\d+/\d+\n}
```

Comment:
I'd like us to get out of the pattern of testing the exit codes first. They're the least informative of our assertions.